### PR TITLE
Scale fix

### PR DESCRIPTION
**Pull request by brianosburn-intel**
fixed UI 404 and tags/hostname issue

### CODE REVIEW

File: scripts/kubernetes/serversChart/nginx/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,21 @@
+server {
```
Comment (rsdmike) [inline scripts/kubernetes/serversChart/nginx/nginx.conf:1]: can we leverage the one in sample-web-ui?

File: scripts/kubernetes/serversChart/templates/mps_webDeployment.yaml [hunk 2]
```diff
@@ -52,6 +52,10 @@ spec:
             value: "{{ .Values.mps.enableLogging }}"
           - name: "MPS_DEBUG"
             value: "{{ .Values.mps.debug }}"
+          - name: "MPS_USE_DB"
+            value: "{{ .Values.mps.useDB }}"
+          - name: "MPS_CONNECTION_STRING"
+            value: "postgresql://{{ .Values.backenddb.dbUser }}:{{ .Values.backenddb.dbPassword }}@{{ .Values.rps.dbHost }}:{{ .Values.rps.dbPort }}/mpsdb"
```
Comment (rsdmike) [inline scripts/kubernetes/serversChart/templates/mps_webDeployment.yaml:58]: just double checking that for RPS -- you've updated the connection strings to reflect `/rpsdb` ?

**Review by rsdmike — CHANGES_REQUESTED** [review]

File: scripts/kubernetes/serversChart/nginx/nginx.conf (hunk 1 above)
Comment (brianosburn-intel) [reply scripts/kubernetes/serversChart/nginx/nginx.conf:1]: was having difficulties using a relative path to the config during deployment plus i was thinking it creates a dependency on that repo. does the config change often?

File: scripts/kubernetes/serversChart/templates/mps_webDeployment.yaml (hunk 2 above)
Comment (brianosburn-intel) [reply scripts/kubernetes/serversChart/templates/mps_webDeployment.yaml:58]: i haven't changed it from what it was.. the init.sql doesn't name the db like initMPS.sql does.

File: scripts/kubernetes/serversChart/nginx/nginx.conf (hunk 1 above)
Comment (rsdmike) [reply scripts/kubernetes/serversChart/nginx/nginx.conf:1]: I think the dependency is fine

File: scripts/kubernetes/serversChart/templates/mps_webDeployment.yaml (hunk 2 above)
Comment (rsdmike) [reply scripts/kubernetes/serversChart/templates/mps_webDeployment.yaml:58]: Right, it's specified in the Postgres env , it should match what we have in the compose. So just making sure that's set to rpsdb, as well as connection string

File: scripts/kubernetes/serversChart/nginx/nginx.conf (hunk 1 above)
Comment (rsdmike) [reply scripts/kubernetes/serversChart/nginx/nginx.conf:1]: https://github.com/open-amt-cloud-toolkit/open-amt-cloud-toolkit/issues/35

**Review by rsdmike — APPROVED** [review]

**Pull request merged**